script.service.ltvpl/addon.xml

```diff
@@ -0,0 +1,57 @@
+<addon id="script.service.ltvpl" name="Live TV Playlist" provider-name="jmooremcc" version="1.0.0">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0" />
+    <import addon="xbmc.addon" version="17.0.0" />
+    <import addon="xbmc.gui" version="5.14.0" />
+  </requires>
+  <extension library="service.py" point="xbmc.service" />
```

Comment:
this is the main addon type. Maybe you should consider moving the script extension point to the top (because it will  be listed under programs in the addon browser)

Reply:
Moved to top